setup.py

```diff
@@ -10,9 +10,11 @@
 
 
 def get_long_description() -> str:
+    changes_md = CURRENT_DIR / "CHANGES.md"
     readme_md = CURRENT_DIR / "README.md"
-    with open(readme_md, encoding="utf8") as ld_file:
-        return ld_file.read()
+    with readme_md.open("r", encoding="utf8") as rmd_file:
+        with changes_md.open("r", encoding="utf8") as cmd_file:
+            return rmd_file.read() + "\n\n" + cmd_file.read()
```

Comment:
because you've got pathlib you can just do:
```python
def get_long_description() -> str:
    return "\n\n".join(
        (CURRENT_DIR / p).read_text(encoding="utf8")
        + "\n\n"
        + (CURRENT_DIR / "CHANGES.md").read_text(encoding="utf8")
    )
```

Reply:
or this if you don't like the duplication:

```python
def get_long_description() -> str:
    return "\n\n".join(
        [
            (CURRENT_DIR / p).read_text(encoding="utf8")
            for p in ["README.md", "CHANGES.md"]
        ]
    )
```